python/Makefile

```diff
@@ -1,3 +1,4 @@
+SHELL := /bin/bash
```

Comment:
How about 
```suggestion
SHELL := /bin/sh
```

Reply:
Won't work, because `source` is a bash specific command which is not available in sh